test/functional/comment_controller_test.rb

```diff
@@ -42,6 +42,31 @@ def teardown
     assert_template partial: 'questions/_comment'
   end
 
+  test 'should create wiki comments' do
```

Comment:
I think this is pulled in from another PR -- but it's already merged, so not sure why its showing up here: https://github.com/publiclab/plots2/pull/1444/files -- but I'll double check before a final merge in to be sure.